lib/seam/resources/seam_error.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Seam
+  class SeamError < BaseResource
```

Comment:
Otherwise it's not consistent with the `Warning`. (See next comment)

Will also need to rename file / update require / update refs.

```suggestion
  class ResourceError < BaseResource
```